packages/ppxx/ppxx.1.2.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+version: "1.2.1"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+bug-reports: "jun.furuse@gmail.com"
+dev-repo: "hg://https://bitbucket.org/camlspotter/ppxx"
+build: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-build" ]
+]
+install: [
+  [ "ocaml" "setup.ml" "-install" ]
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind"
+  "omake"
+  "ppx_tools"
+]
+available: [
+  ocaml-version = "4.02.3"
```

Comment:
If this package works at all with OCaml 4.02.0 or might work with OCaml 4.03.x, it would be great to set this constraint to be more lenient.